### PR TITLE
Internal:  Testing Finder using PHPUnit [ED-6095]

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	/**
+	 * Test if the register hook is running as expected.
+	 */
+	public function test_register__hook() {
+		// Arrange
+		$categories_manager = new Categories_Manager();
+
+		$mock = $this->getMockBuilder( static::class )
+			->setMethods( [ 'on_finder_category_register' ] )
```
Comment (udidol) [inline tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php:63]: What is this method?

File: tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+				$this->isInstanceOf( Categories_Manager::class )
+			);
+
+		// Act
+		add_action( 'elementor/finder/register', [ $mock, 'on_finder_category_register' ] );
+
+		// Trigger the hook.
+		$categories_manager->get_categories();
+	}
+}
```
Comment (udidol) [inline tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php:79]: Add an empty line at the end of each file

Comment (ManorHazaz) [reply tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php]: Done.

File: tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php (hunk 1 above)
Comment (ManorHazaz) [reply tests/phpunit/elementor/core/common/modules/finder/test-categories-manager.php:61]: The test is about the hook. The method is here to make you sure the hook is running once.
The name of the method is irrelevant, you can name it as you wish as long as you calling this method in the hook.

**Review by udidol — DISMISSED** [review]

**Pull request merged**